jackal_base/scripts/calibrate_compass

```diff
@@ -55,12 +56,13 @@ fi
 
 if [[ -r $CAL_FILE ]]; then
   echo "Calibration generated in $CAL_FILE."
-  read -r -p "Copy calibration to /etc/ros/${ROS_DISTRO}? [Y/n] " response
+  read -r -p "Copy calibration to ${CAL_FINAL_PATH}? [Y/n] " response
   response=${response,,}    # tolower
   if [[ $response =~ ^(no|n)$ ]]; then
```

Comment:
http://stackoverflow.com/questions/1728683/case-insensitive-comparision-of-strings-in-shell-script ?